<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="341343" started-at="2020-05-23T13:45:21Z" finished-at="2020-05-23T13:51:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="341343" started-at="2020-05-23T13:45:21Z" finished-at="2020-05-23T13:51:02Z">
      <class name="com.testcases.ShopingTC2">
        <test-method status="PASS" signature="init()[pri:0, instance:com.testcases.ShopingTC2@96def03]" name="init" is-config="true" duration-ms="18782" started-at="2020-05-23T19:15:21Z" finished-at="2020-05-23T19:15:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="shopping()[pri:0, instance:com.testcases.ShopingTC2@96def03]" name="shopping" duration-ms="312887" started-at="2020-05-23T19:15:40Z" finished-at="2020-05-23T19:20:53Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	Total cost is not same on Order Summary page ! expected [$33.02] but found [$35.02]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	Total cost is not same on Order Summary page ! expected [$33.02] but found [$35.02]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.testcases.ShopingTC2.shopping(ShopingTC2.java:253)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- shopping -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.testcases.ShopingTC2@96def03]" name="tearDown" is-config="true" duration-ms="1000" started-at="2020-05-23T19:21:01Z" finished-at="2020-05-23T19:21:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testcases.ShopingTC2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
